packages/makeup-menu-button/dist/mjs/index.js

```diff
@@ -112,8 +113,8 @@ function _onMenuItemSelect(e) {
   if (e.detail.el.getAttribute("role") !== "menuitemradio") {
     return;
   }
-  const icon = e.detail.el.querySelector(this._options.iconSelector).cloneNode(true);
-  const text = e.detail.el.innerText.trim();
+  const icon = e.detail.el.querySelector(this._options.menuItemIconSelector).cloneNode(true);
+  const text = this._options.menuItemButtonLabelSelector ? e.detail.el.querySelector(this._options.menuItemButtonLabelSelector)?.innerText.trim() : e.detail.el.innerText.trim();
   let content = this._buttonPrefix ? `${this._buttonPrefix} ${text}` : text;
   if (icon) {
     switch (this._options.buttonValueType) {
```

Comment:
This code might be easier to follow if we move the type "text" logic within the switch statement too...?